wrappers/vroom.rb

```diff
@@ -62,7 +73,7 @@ def solve_synchronous?(_vrp)
     end
 
     def solve(vrp, job = nil, _thread_proc = nil)
-      if vrp.points.empty? || vrp.services.empty?
+      if vrp.points.empty? || vrp.services.empty? && vrp.shipments.empty?
```

Comment:
Isn't this in fact already managed by OptimiezWrapper::solve ?

Reply:
Use Wrappers::Wrapper.new.empty_result() ?

Reply:
the solve verification is done before the call of detect_unfeasible_services and check_distances for the particular sub problem

Reply:
And yes, the empty_result method should be used here